libraries/api-extractor/src/test/DocItemLoader.test.ts

```diff
@@ -19,6 +19,11 @@ function testErrorHandler(message: string, fileName: string, lineNumber: number)
   capturedErrors.push({ message, fileName, lineNumber });
 }
 
+function assertCapturedErrors(expectedMessages: string[]): void {
+  assert.deepEqual(capturedErrors.map(x => x.message), expectedMessages,
```

Comment:
The behavior is deterministic, so yes they should appear in the expected order.  I could also be validating the filenames and line numbers, but that was too much typing.



---
In reply to: [122826879](https://github.com/Microsoft/web-build-tools/pull/241#discussion_r122826879) [](ancestors = 122826879)